go.mod

```diff
@@ -4,26 +4,14 @@ go 1.16
 
```

Comment:
Shouldn't `sigs.k8s.io/controller-runtime` have been updated to version `v0.9.2` in go.mod?  That's what got generated when used the latest operator sdk to ran `operator-sdk init ...` for the argocd-operator.

Reply:
Got import errors when I tried to upgrade. I think it's because of dependency on `argocd-operator v0.0.16` which uses `sigs.k8s.io/controller-runtime v0.6.0`. Should we fix this in a different PR once the upstream operator is upgraded?

Reply:
I'd agree to fix this after upstream's sdk is updated.